test/integration/authz_v1_test.go

```diff
@@ -119,9 +120,6 @@ func TestCreateServiceRoleOnly(t *testing.T) {
 	}
 	r := fixtures.GetExpectedRbac(o)
 	rolloutAndValidateRbac(t, r, create)
-	namespace := athenz.DomainToNamespace(r.AD.Name)
```

Comment:
Why is this part removed?

Reply:
+1

Reply:
In this case we still end up creating a ServiceRoleBinding containing the  [Spiffe URI](https://github.com/yahoo/k8s-athenz-istio-auth/blob/464d0c40214dfb289bc1b8f0eaa079701baf6235/test/integration/fixtures/fixtures.go#L250-L252), so the test case is no longer valid.